script.js

```diff
@@ -97,45 +125,82 @@ function setPlatform(platform) {
     currentPlatform = platform;
 }
 
-function generateRegionInfo(name, ip, fqdn, port) {
+function setPortIfDefault(protocol) {
+    const oldPort = protocol == "http" ? "443" : "22000";
+    const newPort = protocol == "http" ? "22000" : "443";
```

Comment:
Use the constant `DEFAULT_PORT` here? (and split it to `DEFAULT_HTTP_PORT` and `DEFAULT_HTTPS_PORT` I guess)

Reply:
yeah, that'd be cleaner, I'll change it